website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+* the `id` MUST contain the globally unique member identifier as maintained in the register.  Often this will be a web link to the member entry.
+* `name` is the human readable registered entity name and SHOULD match the registered name as it appears in the authoritative register.
+* `registeredId` contains the simple identifier value that is unique within the register (but may not be globally unique) for example the VAT registration number.
+* `idScheme` identifies the authoritative register.  If the identity scheme is registered with UN/CEFACT then the `idScheme.id` MUST match the `identityRegister.id` in the UN/CEFACT scheme register.
+* `registerType` is a coded value that allows verifiers to distinguish between different DIA [use cases](#use-cases)
+* `verifiedDIDList` contains an array of DIDs that the member has requested (and the registrar has verified) to be linked to the registeredId of the member.
+* `registrationScopeList` contains a list of URIs that define the scope of the member registration. The values are very specific to the register. For example a national business register would typically have a controlled vocabulary of entity types (eg [Australian Entity Types](https://abr.business.gov.au/Help/EntityTypeList))
+
+
+```json
+  "credentialSubject": {
+    "type": [
+      "RegisteredIdentity",
+      "Identifier"
+    ],
+    "id": "https://sample-register.gov/90664869327",
+    "name": "Sample business Ltd",
+    "registeredId": "90664869327",
+    "idScheme": {
+      "type": [
+        "IdentifierScheme"
+      ],
+      "id": "https://sample-register.gov",
+      "name": "Sample National Business Register"
+    },
+    "registerType": "Business",
+    "verifiedDIDList": [
+      "did:web:samplecompany.com/123456789",
+      "did:ethr:0x5:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74"
+    ],
+    "registrationScopeList": [
+      "https://sample-register.gov/EntityType?Id=00019"
+    ]
+  }
+```
+
+## DIA Discovery
+
+DIA credentials SHOULD be discoverable from either identifier:
+
+* Given a DID (eg as the issuer of a DPP) via DID document `service` end point.
+* Given a registered identifier (eg a VAT registration number) via the ID scheme resolver service.
+
+### Via DID Service EndPoint
+
+As described in the [W3C Decentralized Identifiers](https://www.w3.org/TR/did-core/) specification, DIDs are resolvable to a DID document. The `service` property of a DID document contains an array of typed `serviceEndpoint` which can point to services or credentials relevant to the DID. A DID document may also contain an "alsoKownAs" property which is typically used to reference other identifiers. Controllers of DIDs that are linked to authoritative register SHOULD
+
+* Add the ID URI from the authoritative register to the `alsonKnownAs` list. In the snippet below `https://sample-register.gov/90664869327` has been added.
```

Comment:
I'm not sure I understand this. Not all governments offer a direct link to a registration, often time it will be a query service. I'd put this as an optional step.